tesla_ver/bubble_chart/bubble_chart.py

```diff
@@ -47,6 +47,11 @@ def extract_mdata(df, x_column_name):
                 "time_max": max(x_vals),
                 "x_vals": x_vals,
                 "data_cols": [col for col in df.columns.values.tolist() if col not in ["X", "Year", "Subject"]],
+                "ranges": {
+                    col: (min(df[col]) * 1.25, max(df[col] * 1.25))
+                    for col in df.columns.values.tolist()
+                    if col not in ["X", "Year", "Subject"]
```

Comment:
What happens if there is a non-numeric column with a header different from these?

Reply:
Right now, those values are hardcoded, and the data files need to fit that pattern -- I'm not thinking of a good way besides either having the user rename the columns later, or some particularly large regex to try and guess at what the ID and time values are.